test/unit/cart.js

```diff
@@ -67,7 +70,8 @@ describe('Cart class', () => {
             title: 'test',
             variant_title: 'test2',
             line_price: 20,
-            quantity: 1
+            quantity: 1,
+            variant: { image: { src: 'cdn.shopify.com/image.jpg' } }
```

Comment:
ah. i always default to adding spaces, and then fix it when the linter yells at me. i'm surprised it didn't catch this.